src/SOS/SOS.Hosting/DataTargetWrapper.cs

```diff
@@ -485,5 +513,14 @@ private delegate int GetRuntimeBaseDelegate(
             [Out] out ulong address);
 
         #endregion
+
+        #region ICLRContractLocator delegate
+
+        [UnmanagedFunctionPointer(CallingConvention.Winapi)]
+        private delegate int GetContractDescriptorDelegate(
+            [In] IntPtr self,
+            [Out] out ulong address);
```

Comment:
The `In` and `Out` attributes aren't needed.

Reply:
They are on the rest of the delegates in this file and they make it clearer what parameters are in/out.